schema.sql

```diff
@@ -9,5 +9,23 @@ CREATE TABLE animals (
     weight_kg decimal(5,2)
 );
 
+--2
+
+ALTER TABLE animals
+ADD species VARCHAR(100);
+
+--3
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
 ALTER TABLE animals
-ADD species VARCHAR(100);
\ No newline at end of file
+ADD COLUMN species_id integer REFERENCES species(id),
+ADD COLUMN owner_id integer REFERENCES owners(id);
```

Comment:
- [ ] Ouch! this is a close one but please make sure to drop the `species` column before adding the above 2 columns as specified in the project requirements. 💯